client/app/components/packages/pas-form/show.hbs

```diff
@@ -465,7 +465,7 @@
         </A.Prompt>
         <A.Field>
           <span data-test-proposedDevelopmentTypes>
-            <ProposedDevelopmentTypesList @pasForm={{@package.pasForm}} />
+            <Packages::ProposedDevelopmentTypesList @pasForm={{@package.pasForm}} />
```

Comment:
Could this go in `<Packages::PasForm::ProposedDevelopmentTypesList>` <-- this seems quite unique, not sure if we'd reuse this